great_expectations/data_context/data_context/base_data_context.py

```diff
@@ -18,17 +17,12 @@
 )
 
 
-@deprecated_argument(argument_name="ge_cloud_mode", version="0.15.37")
-@deprecated_argument(argument_name="ge_cloud_config", version="0.15.37")
-def BaseDataContext(  # noqa: PLR0913
```

Comment:
Not supporting `project_root_dir` for `BaseDataContext` or `DataContext` - lmk if you disagree